--- src-gen/backend/server.js       2015-10-21 07:28:03.14159265 +0000
+++ src-gen/backend/server.js   1985-11-18 09:00:03.14159265 +0000
@@ -33,8 +33,8 @@
     const cliManager = container.get(CliManager);
     return cliManager.initializeCli(argv).then(function () {
         const application = container.get(BackendApplication);
-        application.use(express.static(path.join(__dirname, '../../lib')));
-        application.use(express.static(path.join(__dirname, '../../lib/index.html')));
+        application.use(express.static('lib'));
+        application.use(express.static('lib/index.html'));
         return application.start(port, host);
     });
 }